recipe/cmake-configure-alignment.patch

```diff
@@ -0,0 +1,63 @@
+--- CMakeLists.txt	2019-10-01 18:19:12.000000000 +0200
++++ CMakeLists.txt	2020-04-19 10:39:47.000000000 +0200
+@@ -98,6 +98,7 @@
+ SET(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake) # for FindReadline.cmake
+ 
+ SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -I${PROJECT_SOURCE_DIR}/src")
++SET(PACKAGE_VERSION 10.34)
+ 
+ # external packages
+ FIND_PACKAGE( BZip2 )
+@@ -540,16 +541,36 @@
+ # 8-bit library
+ 
+ IF(PCRE2_BUILD_PCRE2_8)
+-ADD_LIBRARY(pcre2-8 ${PCRE2_HEADERS} ${PCRE2_SOURCES} ${PROJECT_BINARY_DIR}/config.h)
+-SET_PROPERTY(TARGET pcre2-8
+-  PROPERTY COMPILE_DEFINITIONS PCRE2_CODE_UNIT_WIDTH=8)
++  ADD_LIBRARY(pcre2-8 ${PCRE2_HEADERS} ${PCRE2_SOURCES} ${PROJECT_BINARY_DIR}/config.h)
++
++SET_TARGET_PROPERTIES(pcre2-8 PROPERTIES
++  COMPILE_DEFINITIONS PCRE2_CODE_UNIT_WIDTH=8
++  MACHO_COMPATIBILITY_VERSION 10.0.0
++  MACHO_CURRENT_VERSION 10.0.0
++  VERSION 0.9.0
++  SOVERSION 0)
```

Comment:
Shouldn't all these versions be parsed from `configure.ac` like `PCRE2_MAJOR` is parsed just a few lines above in the CMakeLists.txt

Reply:
Refreshed my autotools version computation knowledge and all versions are now parsed from `configure.ac`.